### PR TITLE
Feature/86cv4duft Booking Summary Page

### PR DESCRIPTION
**Pull request by nethbtju**
This file is accessed through http://localhost:3000/request-booking. Input your booking request details and click next step to make summary appear.

Please double check the profilePhoto stuff works cos my git decided to obliterate it from existence...

### CODE REVIEW

**Review by joshualoongwy — CHANGES_REQUESTED** [review]

File: behind-the-veil-siteroot/imports/ui/components/toolTip/ToolTip.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import "./tooltip.css";
+
+
+/**
+ * ToolTip component displays a tooltip with an icon and text.
+ * @param {JSX.Element} tipIcon - The icon for the tooltip.
+ * @param {string} tipText - The text to be displayed in the tooltip.
+ * @returns {JSX.Element} - The rendered ToolTip component.
+ */
+const ToolTip = ({ tipIcon: ToolTipIcon, tipText}) => {
```
[the remark this answers is not in the document]
Comment (nethbtju) [reply behind-the-veil-siteroot/imports/ui/components/toolTip/ToolTip.js:17]: Yeah not bad of an idea I'll look into it rn

File: behind-the-veil-siteroot/imports/ui/components/pages/request-booking/booking.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.label {
```
[the remark this answers is not in the document]
Comment (nethbtju) [reply behind-the-veil-siteroot/imports/ui/components/pages/request-booking/booking.css:1]: oh completely missed that. Kinda suck at tailwind rip

File: behind-the-veil-siteroot/imports/ui/components/toolTip/ToolTip.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (nethbtju) [reply behind-the-veil-siteroot/imports/ui/components/toolTip/ToolTip.js]: i added this, can you have a look it works fine on your end?

File: behind-the-veil-siteroot/imports/ui/components/pages/home/HomePage.jsx [hunk 3]
```diff
@@ -144,34 +144,27 @@ export const HomePage = () => {
...
-                <div className="absolute rounded-full z-[19] blur-md
-                    border-main-blue border-2
-                    bg-gradient-to-r from-bg-gradient-end/85 to-bg-gradient-start/85
-                    left-[16%] top-[42.5%] w-[68%] h-[12%]
-                    sm:top-[54%] sm:h-[12%] sm:blur-md
-                    md:top-[52.5%] md:h-[15%] md:blur-lg
-                    lg:top-[46.5%] lg:h-[22%] lg:blur-xl
-                    xl:top-[47.5%] xl:h-[25%] xl:blur-2xl">
-                </div>
+            <div
```
Comment (joshualoongwy) [inline behind-the-veil-siteroot/imports/ui/components/pages/home/HomePage.jsx:147]: oh btw, just wondering are we going with the fuzzier design in figma? this is what this branch looks like
image

File: behind-the-veil-siteroot/imports/ui/components/toolTip/ToolTip.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (joshualoongwy) [reply behind-the-veil-siteroot/imports/ui/components/toolTip/ToolTip.js]: yep tooltip looks fine to me

File: behind-the-veil-siteroot/imports/ui/components/pages/request-booking/FormOutput.jsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    return (
+        <div className="flex text">
+            <span className={`label min-w-[150px] lg:w-[200px] ${textColor} font-semibold`}>
+                {label}
+            </span>
+            <span className="input font-semibold">
+                {input}
+            </span>
+            {haveHelpText && (
+                <span className="tooltip content-center left-1 ml-2">
```
Comment (joshualoongwy) [inline behind-the-veil-siteroot/imports/ui/components/pages/request-booking/FormOutput.jsx:30]: i dont think the classes 'tooltip' and 'left-1' do anything here

File: behind-the-veil-siteroot/imports/ui/components/pages/request-booking/FormOutput.jsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+ * @param {string} input - The input value.
+ * @param {string} textColor - The color of the label text.
+ * @param {boolean} haveHelpText - Indicates if the tooltip should be displayed.
+ * @param {string} tipText - The text to be displayed in the tooltip.
+ * @returns {JSX.Element} - The rendered FormOutput component.
+ */
+const FormOutput = ({ label, input, textColor, haveHelpText, tipText}) => {
+    return (
+        <div className="flex text">
+            <span className={`label min-w-[150px] lg:w-[200px] ${textColor} font-semibold`}>
```
Comment (joshualoongwy) [inline behind-the-veil-siteroot/imports/ui/components/pages/request-booking/FormOutput.jsx:23]: the class 'label' doesn't do anything anymore so can remove it

**Review by joshualoongwy — APPROVED** [review]

**Pull request merged**